trace-test-application/build.gradle

```diff
@@ -36,7 +36,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "io.bitrise.trace:trace-sdk:${traceVersions.traceSdk}"
+    implementation project(path: ':trace-sdk')
```

Comment:
this was an Android Studio suggestion because it couldn't pick up the TraceSdk change without publishing to mavenLocal. I don't think it should cause any issues - and it should make things faster to test locally :-)